cmd/kubeadm/app/cmd/util/cmdutil_test.go

```diff
@@ -17,6 +17,10 @@ limitations under the License.
 package util
 
 import (
+	"errors"
+	"github.com/spf13/cobra"
+	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
+	"strings"
```

Comment:
these imports are not cortrectly organized.

should be:

stdlib
external like github
k8s.io (non-kubeadm)
k8s.io kubeadm

separated by new line

Reply:
fixed